src/resources/js/tickets.js

```diff
@@ -193,879 +132,962 @@ var ticketHeaderImage = window.ticketHeaderImage || {};
 		} );
 	}
 
-	$( document ).ready( function() {
-		if ( $event_pickers.length ) {
-			startofweek = $event_pickers.data( 'startofweek' );
+	/**
+	 * Returns the currently selected default ticketing provider.
+	 * Defaults to RSVP if something fails
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	function get_default_provider() {
+		var $checked_provider = $( 'input[name=default_ticket_provider]', '#tribe_panel_settings' ).filter( ':checked' );
+		return ( $checked_provider.length > 0 ) ? $checked_provider.val() : 'Tribe__Tickets__RSVP';
+	}
+
+	/**
+	 * Sets the ticket edit form provider to the currently selected default ticketing provider.
+	 * Defaults to RSVP if something fails
+	 *
+	 * @since TBD
+	 *
+	 * @param boolean force selection to RSVP
+	 * @return void
+	 */
+	function set_default_provider_radio( force_rsvp ) {
+		force_rsvp = undefined === force_rsvp ? false : true;
+		var $checked_provider = $( 'input[name="default_ticket_provider"]', '#tribe_panel_settings' ).filter( ':checked' );
+		var provider_id = 'Tribe__Tickets__RSVP_radio';
+
+		if ( ! force_rsvp && $checked_provider.length > 0 ) {
+			provider_id = $checked_provider.val() + '_radio';
+		 }
+
+		$( document.getElementById( provider_id ) ).prop( 'checked', true ).trigger('change');
+	}
+
+	/**
+	 * Returns the current global capacity (via the settings panel.
+	 *
+	 * @since TBD
+	 *
+	 * @return string
```

Comment:
Hate to be a pedant - but this could also return type `number` (looking at the last line of the function, we can return `0` depending on how the ternary is evaluated)